src/cclib/parser/daltonparser.py

```diff
@@ -501,6 +515,9 @@ def extract(self, inputfile, line):
             self.set_attribute("charge", int(line.split()[-1]))
         if "@    Spin multiplicity and 2 M_S" in line:
             self.set_attribute("mult", int(line.split()[-2]))
+        # Dalton only has ROHF, no UHF
+            if self.mult != 1:
+                self.metadata["spintype"] = "ROHF"
```

Comment:
That would be fine, since these are two more or less independent things. That might be a nice thing to call `spintype` (although `open_shell` is fine, too).